src/app/editor.js

```diff
@@ -23,7 +23,9 @@ var css = csjs`
     width             : 100%;
   }
 `
-document.querySelector('#editor-container').className = css['editor-container']
+setTimeout(_ => {
```

Comment:
because the refactoring is not done yet. this is what i'm working on right now and it has a lot of implications and quite a lot of code changes.

Currently, the app tend's toward "spagetthi code" in the sense that, no matter where, there are `document.querySelectors` all over the place grabbing random stuff in the DOM and changing it.

The first step of trying to get rid of it was to clear `index.html` from stuff, but now, all kinds of components can't grab nodes with `document.querySelector`, so delaying there actions with `setTimeout` will make them still work - for now.

First goal for now is to shuffle lots of code into it's own components or add them to existing ones, removing code from `app.js` with the goal of removing the `setTimeout`